packages/babel-traverse/src/context.js

```diff
@@ -105,6 +105,9 @@ export default class TraversalContext {
       path.resync();
       path.pushContext(this);
 
+      // this path is no longer belongs to the tree
+      if (path.key === null) continue;
```

Comment:
What causes this to be set to null? we might want to address that instead of doing it here